### PR TITLE
updating links to docs and removing stuff that is allready in the docs

### PR DESCRIPTION
**Pull request by iruzevic**

### CODE REVIEW

File: Gutenberg/General.md [hunk 1]
```diff
@@ -6,32 +6,6 @@ All development details can be found on:
 - [Github](https://github.com/WordPress/gutenberg/)
 - [Handbook](https://developer.wordpress.org/block-editor/)
```
Comment (dingo-d) [inline Gutenberg/General.md:8]: I know that's in the docs, but I don't see the value of removing this from the handbook. Handbook will just give a high level, a quick overview of why we do things as we do, which is IMO better for new developers then diving into the technical docs right away.

File: Useful links/Useful links.md [hunk 2]
```diff
@@ -4,37 +4,38 @@ Here are some useful links that will help you during the onboarding, but will al
 
 [WordPress Developer Reference](https://developer.wordpress.org/reference/)
 
-[WordPress Core Handbook](https://make.wordpress.org/core/handbook/)  
-[WordPress Themes Handbook](https://make.wordpress.org/themes/handbook/)  
-[WordPress Plugins Handbook](https://make.wordpress.org/plugins/handbook/)  
-[Gutenberg Documentation](https://developer.wordpress.org/block-editor/)  
+[WordPress Core Handbook](https://make.wordpress.org/core/handbook/)
```
Comment (dingo-d) [inline Useful links/Useful links.md:7]: The two spaces at the end are there on purpose, this is how markdown knows to put things in the new line, please revert this change 🙂

File: Gutenberg/General.md (hunk 1 above)
Comment (iruzevic) [reply Gutenberg/General.md:8]: I would like to avoid on having the same thing on two places that is why I have removed it from the handbook.

File: Useful links/Useful links.md (hunk 2 above)
Comment (iruzevic) [reply Useful links/Useful links.md:7]: Oh sorry I will revert this

File: Gutenberg/General.md (hunk 1 above)
Comment (gabriel-glo) [reply Gutenberg/General.md:8]: I'd leave this in. Handbook is something that new people read first, so it's a good general intro to what they'll be doing.

Comment (dingo-d) [reply Gutenberg/General.md:8]: I agree with Gabriel. Since this is the first thing new developers read, it will bring some high-level overview of our work.

Eightshift docs are more in-depth and dev oriented IMO.

**Comment by iruzevic** [comment]
I have returned the changes
